php/Terminus/Models/Collections/Workflows.php

```diff
@@ -90,9 +89,12 @@ protected function getFetchUrl() {
   /**
    * Names the model-owner of this collection
    *
-   * @return [string] $owner_name
+   * @return [string] $this->owner_type or $owner_name
    */
   protected function getOwnerName() {
```

Comment:
This function is bogus. I think we might just start fresh with `getOwnerType()` and have it clearly reference the attribute.